wcivf/apps/elections/templates/elections/includes/_single_ballot.html

```diff
@@ -59,9 +59,8 @@ <h2>{% if postelection.is_london_assembly_additional %}Additional members{% else
               {% endif %}
             {% endif %}
 
-            {% if postelection.get_voting_system %}
-              {% include "elections/includes/_how-to-vote.html" with voting_system=postelection.get_voting_system %}
-            {% endif %}
+            {% include "elections/includes/_how-to-vote.html" with voting_system=postelection.get_voting_system %}
```

Comment:
If we're going to rely on this existing in the code, we should also make voting system required at the database level.

I don't think that's for this PR, but could you make an issue to change [this line](https://github.com/DemocracyClub/WhoCanIVoteFor/blob/master/wcivf/apps/elections/models.py#L36) to make it not null?

We'll also need to make sure the importer always adds the voting system at import time. It looks like this is a good chance to improve performance of the importer, as it's [looking up the voting system for each ballot](https://github.com/DemocracyClub/WhoCanIVoteFor/blob/e0d842ac3365fa68702afa4173365fef49372b5b/wcivf/apps/elections/import_helpers.py#L76) at the moment.